### PR TITLE
Fix interactive option, add decoration option, add epsg4326

### PR DESCRIPTION
**Pull request by JulianBerger**
Fixed the interactive option, I hope the way I did it is okay?!

Added a decoration option for the map container.

Added EPSG4326 projection the second time since the last time it conflicted with another pull request.

### CODE REVIEW

File: lib/src/layer/tile_layer.dart [hunk 1]
```diff
@@ -53,6 +53,9 @@ class TileLayerOptions extends LayerOptions {
   ///Color shown behind the tiles.
   final Color backgroundColor;
 
+  ///Container Widget decoration
+  final Decoration decoration;
```
Comment (johnpryan) [inline lib/src/layer/tile_layer.dart]: Why is this required? If you wanted decoration around the map, couldn't the map widget be wrapped in a Container?

Comment (JulianBerger) [reply lib/src/layer/tile_layer.dart]: I added it to test some styling but you are right, its probably unnecessary.

File: lib/src/map/flutter_map_state.dart [hunk 2]
```diff
@@ -52,11 +52,11 @@ class FlutterMapState extends MapGestureMixin {
           .map((layer) => _createLayer(layer, widget.options.plugins))
           .toList();
       return new GestureDetector(
-        onScaleStart: handleScaleStart,
-        onScaleUpdate: handleScaleUpdate,
-        onScaleEnd: handleScaleEnd,
+        onScaleStart: options.interactive ? handleScaleStart : null,
+        onScaleUpdate: options.interactive ? handleScaleUpdate : null,
```
Comment (johnpryan) [inline lib/src/map/flutter_map_state.dart]: Can you help me understand why this is required? I thought that if `handleScaleStart` is null the map will behave normally. I don't see why `interactive` is required for the API.

**Comment by johnpryan** [comment]
fixed by #207 - can you make a separate PR for epsg4326? It's easier to get things merged when they are smaller changes.
